README.md

```diff
@@ -114,13 +114,17 @@ See the [annotation's implementation](core/src/main/scala/com/github/sbt/jni/ann
 
 #### Usage example (Scala 3.x / Scala 2.x):
 
-Scala 3 has no macro annotations support. As a solution we don't need this to be a macro function anymore. As the result, this option requires to have an explicit dependency on the [core](./core) sub project.
+Scala 3 has no macro annotations support. As a solution we don't need this to be a macro function anymore. As the result, this option requires to have an explicit dependency on the [sbt-jni-core](./core) library. If you want to run or test the project from sbt and have `ThisBuild / turbo := true`, you have to change the `classLoaderLayeringStrategy` to `ClassLoaderLayeringStrategy.Flat`, otherwise you will get `UnsatisfiedLinkError`, like `java.lang.UnsatisfiedLinkError: 'int simple.Adder.plus(int)'`.
```

Comment:
Turbo mode feels like a separate paragraph, below the default configuration. Can be also nice to add links to SBT https://www.scala-sbt.org/1.x/docs/sbt-1.3-Release-Notes.html#Turbo+mode+with+ClassLoader+layering.

Reply:
Btw it's relevant to `NativeLoader`, it doesn't happen with `@nativeLoader`

Reply:
@sideeffffect which makes sense, because with `@nativeLoader` there are no extra Compile time deps :+1: